Return the response after sending the message

pavoltanuska opened the pull request:
Laravel's notification sender (`Illuminate\Notifications\NotificationSender`) dispatches an event (`Illuminate\Notifications\Events\NotificationSent`) after the message has been sent.
This event contains the response returned from the specific notification channel. When no response is returned, this event's response attribute contains `null`.

However, this information could be useful - for logging purposes, for pairing the notification messages with the users etc.

LKDevelopment commented:
First of all thank you @pavoltanuska for the contribution! It looks really great! Is it possible that you add /edit our Tests for the Return types?

pavoltanuska commented:
Thank you both for review, guys. I've added asserts for response to existing tests and added test checking for `null` response in the "player_id empty" case.

@Lloople as you can see, the explicit `null` causes problems with StyleCI analysis. Empty `return;` is implicitly null.

I also see there's another problem with the order of `use` statements.

Should I fix these 2 StyleCI errors?

Lloople commented:
You’re right on that null thing 🙂 you can add another commit fixing both styleCI errors if you want. Let’s wait for @LKDevelopment review before merging 👍

LKDevelopment commented:
Looks good @Lloople can you merge and release this :)? 

-- sended from my mobile Phone
